SD DB Setup.sql

```diff
@@ -422,7 +422,7 @@ create table voting_rights
     voting_notes text,
     pending_leg text,
     date_modified timestamp not null default current_timestamp(),
-    user_modified varchar(20) not null);
+    user_modified varchar(20) default null);
     
```

Comment:
The voting_rights script shouldn't have to provide a user_modified column value upon INSERT. In order to do that without a complaint from MySQL, the column definition needs a default value. I chose NULL for the default, but this could instead be "not null default ''" if you prefer.